### PR TITLE
[SSPROD-8261] add agent presubmit

### PR DESCRIPTION
**Pull request by speedyguy17**
run presubmit against any changes which are not coming direct from upstream

Self-fulfilling PR

### CODE REVIEW

File: scripts/jenkins/presubmit.jenkins [hunk 1]
```diff
@@ -0,0 +1,146 @@
+/* vim:expandtab:sw=4:ts=4:sts=4
+*/
+
+
+def getAgentCommitSha() {
```
Comment (nathan-b) [inline scripts/jenkins/presubmit.jenkins:5]: I think this function might be misnamed. Won't this get the sha for the oss-falco repo?

Comment (speedyguy17) [reply scripts/jenkins/presubmit.jenkins:5]: it will. we can pretend "agent" here is the agent that gets the commit sha? IDK :D or tell me to do my job

Comment (nathan-b) [reply scripts/jenkins/presubmit.jenkins:5]: lol, works for me

**Review by nathan-b — APPROVED** [review]
rubberstamp.png

**Pull request merged**